linux/inc/c_pal/execution_engine_linux.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#ifndef EXECUTION_ENGINE_LINUX_H
+#define EXECUTION_ENGINE_LINUX_H
+
+#ifdef __cplusplus
+#include <cstdint>
+#else
+#include <stdint.h>
+#endif
+
+#include "c_pal/execution_engine.h"
+
+#include "umock_c/umock_c_prod.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
Likely this extern "C" needs to be just before the execution_engine_linux_get_parameters.
The struct and defines don't need to be within it.
/consider only